src/components/molecules/EarthEditor/PropertyPane/PropertyField/URLField/index.tsx

```diff
@@ -1,33 +1,18 @@
-import { useApolloClient } from "@apollo/client";
-import React, { useCallback, useState } from "react";
+import React, { useCallback, useState, ComponentType } from "react";
 import { useIntl } from "react-intl";
 
 import Icon from "@reearth/components/atoms/Icon";
-import AssetModal, {
-  Asset as AssetType,
-  AssetSortType as SortType,
-} from "@reearth/components/molecules/Common/AssetModal";
+import { Props as AssetModalPropsType } from "@reearth/components/molecules/Common/AssetModal";
 import { styled } from "@reearth/theme";
 
 import TextField from "../TextField";
 import { FieldProps } from "../types";
 
-export type Asset = AssetType;
-
-export type AssetSortType = SortType;
+export type AssetModalProps = AssetModalPropsType;
```

Comment:
For a more rigorous typing, you can write as following:

```suggestion
export type AssetModalProps = Pick<AssetModalPropsType, "isOpen" | "fileType" | "initialAssetUrl" | "onSelect" | "toggleAssetModal">;
```